LiveSched/src/main/java/dev/coms4156/project/livesched/Task.java

```diff
@@ -0,0 +1,117 @@
+package dev.coms4156.project.livesched;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+/**
+ * Represents a task that has to be done. 
+ * This class stores the id, resources needed, and the location of the task.
+ */
+public class Task {
+  private final String taskId;
+  private Map<ResourceType, Integer> resourceList; // Key = ResoureceType, Value = Units needed
+  private int priority; 
+  private LocalDateTime startTime;
+  private LocalDateTime endTime;
+  private double latitude;
+  private double longitude;
+
+  /**
+   * Constructs a new Task object with the given parameters.
+   *
+   * @param taskId        the unique ID of the resource
+   * @param resourceList  the map of resources and their quantity needed
+   * @param priority      the priority of the task
```

Comment:
Also, what should be the allowable range for priority? 1 through 5?

Reply:
What should the metric be for the priority? Should we take some information from the client to calculate the priority in the Task class or are we assuming that the client will specify?

Reply:
I think, for now, we can assume that the client will specify!